docs/ci-integration/guides/jenkins.md

```diff
@@ -0,0 +1,93 @@
+# Jenkins
+
+## Example
+
+You can find our [Jenkins example here](https://github.com/earthly/ci-examples/tree/main/jenkins).
+
+To run it yourself, clone the `ci-examples` repository, and then run (from the root of the repository):
+
+```go
+earthly ./jenkins+start
+```
+
+This will start a local Jenkins server, minimally configured to spawn `earthly` builds using the Docker cloud plugin.
```

Comment:
This example is great, but I feel that the fact that this guide starts with the example creates a bit of dissonance. Reason: The reader wants to install Earthly in their existing Jenkins, not in a new Jenkins that Earthly creates.

It might be better if we start with a step-by-step guide about how to install Earthly in an existing Jenkins installation (ideally with pictures, if there's clicking needed) and then at the end to showcase this Jenkins example that creates a Jenkins via Earthly.

Reply:
The example is meant to provide a playground; not a ready-to-go configuration. It has all the parts needed to use Earthly inside it, but isn't fully production ready.

Reply:
Reworked the guide format, and added a guides template.